website/docs/policy/releases.mdx

```diff
@@ -7,6 +7,21 @@ import TierLabel from "../_components/TierLabel";
 
 # Profile Releases <TierLabel tiers="enterprise" />
 
+## v0.6.2
+
+### Highlights
+- **Agent**
+  - Add Terraform mode to allow validating terraform plans
+  - Support targeting kubernetes HPA resources
+
+### Dependency Versions
+
+- policy-agent 1.2.0
+
+### Policy Library Compatibility
```

Comment:
@waleedhammam let's change the compatability section to match this https://www.notion.so/weaveworks/Policy-Agent-v1-2-0-5709e5398bb44d52b387584344c4d0f5#e7f0785dc6b249dcbae5b7330a63a05b and add the same note below